packages/dashboard/src/app/views/savedTemplates/test/savedTemplates.js

```diff
@@ -44,7 +44,9 @@ jest.mock(
       children
 );
 
-describe('<SavedTemplates />', function () {
+// TODO https://github.com/google/web-stories-wp/issues/8746
+// eslint-disable-next-line jest/no-disabled-tests
+describe.skip('<SavedTemplates />', function () {
```

Comment:
this view is tragically out of date at this point. TODO to decide to clean it up or delete it